appknox/client.py

```diff
@@ -110,7 +125,8 @@ def login(self, otp: int = None):
         if otp:
             data['otp'] = str(otp)
 
-        response = requests.post(urljoin(self.host, 'api/login'), data=data)
+        response = requests.post(
+            urljoin(self.host, 'api/login'), data=data, proxies=self.proxies)
```

Comment:
Add self.proxies in the `self.json_api` and `self.drf_api` in the login.

Reply:
fixed